src/compiler/utilities.ts

```diff
@@ -4574,14 +4574,14 @@ namespace ts {
 
         // First try the entire locale, then fall back to just language if that's all we have.
         // Either ways do not fail, and fallback to the English diagnostic strings.
-        if (!trySetLanguageAndTerritory(language, territory, errors)) {
-            trySetLanguageAndTerritory(language, undefined, errors);
+        if (!trySetLanguageAndTerritory(language, territory, sys, errors)) {
+            trySetLanguageAndTerritory(language, /*territory*/ undefined, sys, errors);
         }
 
         return true;
     }
 
-    export function trySetLanguageAndTerritory(language: string, territory: string, errors?: Diagnostic[]): boolean {
+    export function trySetLanguageAndTerritory(language: string, territory: string, sys: System, errors?: Diagnostic[]): boolean {

```

Comment:
Do not think you need the whole thing..

consider : `host:  {getExecutingFilePath():string, readFile(name:string):string; }`